packages/node_modules/@ciscospark/react-component-utils/src/metrics.js

```diff
@@ -0,0 +1,116 @@
+const METRIC_NS = `ciscospark-widget-`;
+const LOAD_START = `startLoad`;
+
+/**
+ * Temporarily save metric to sessionStorage
+ * @param {string} key
+ * @param  {any} data
+ * @returns {null}
+ */
+export function saveTempMetric(key, data) {
+  window.sessionStorage.setItem(key, data);
```

Comment:
Yea, probably a better idea... also easier to deal with in unit tests. I don't have to mock out sessionStorage.